### DISCUSSION TITLE
Maintenance of the project

### DISCUSSION THREAD

**Discussion by yzsolt** [jackmott/simdeez#75] [General]
Hey @arduano,
in the last ~1 year there wasn't any PR merged, including mine (https://github.com/arduano/simdeez/pull/66 and https://github.com/arduano/simdeez/pull/64). I don't want to blame you - open source project maintainership is difficult, especially when one is doing it for free, in one's free time. I'd like to offer my help instead, and spark a general discussion on how the situation could be improved.
I'm doing open source in my free time too, and I'm interested in `simdeez` and [rust-simd-noise](https://github.com/verpeteren/rust-simd-noise) because of a personal hobby project. I'm not a SIMD expert, but I thought I could help with reviews and general project maintenance (releases, CI, improving docs, etc.). I don't want to take over the repository, but I think more people having the ability of merging PRs would be required to move things forward. Maybe @verpeteren would be interested too? We could set up some rules (like at least one of us has to review the other's PRs before merging).
Please share your thoughts! I'd really like to see progress in both `simdeez` and `simd-noise`.

**Comment by arduano** [comment]
Hey, yeah I must've missed your PRs during some of my most stressful university months. Generally I try to be on top of PR reviews for projects I own, but this project slipped through, apologies.

The main issue with this project is that after I took over ownership and did my rewrite, I couldn't figure out how to add `sleef` compatibility back. Basically SIMD-powered trig operations. If anyone finds a way to add those in, I'll happily release a new 2.0 version of simdeez. For now I'm stuck releasing pre-release versions.
